client/driver/docker.go

```diff
@@ -268,7 +261,10 @@ func (d *DockerDriver) createContainer(ctx *ExecContext, task *structs.Task, dri
 
 	// Setup port mapping and exposed ports
 	if len(task.Resources.Networks) == 0 {
-		d.logger.Print("[WARN] driver.docker: No network resources are available for port mapping")
+		d.logger.Print("[INFO] driver.docker: No network interfaces are available")
```

Comment:
This debug message is only interesting if they want port forwarding. If I don't ask for any ports then I don't really care about this message. Also if you do decide to keep it, Print -> Println